themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+---
+title: "Pulumi Service API"
+meta_desc: An overview of the Pulumi Service API for querying Organization, Stack, State, etc. information.
+---
+
+The Pulumi Service API is used by the Pulumi CLI and the Pulumi Console to query and interact with state information, history, stack tags, etc. This API is available for end users to integrate into their own automation use cases.
+
+{{% notes "info" %}}
+While the Pulumi Service API is used by the Pulumi CLI and Pulumi Console, Pulumi makes no guarantees about changes that might affect backwards compatibility or cause breaking changes.
+{{% /notes %}}
+
+## Authentication
+
+All requests must be authenticated using a token via the `Authorization` HTTP header. If the token is missing or invalid, the Pulumi Service API will return a 4xx status code.
+
+The `Authorization` header must be in the form below with the literal string `token`, then a space, then your access token value.
+
+```
+Authorization: token {token}
+```
+
+{{% notes "info" %}}
+As of June 28, 2019, Pulumi access tokens contain the prefix `pul-`, which should be included as part of the token value in the header.
+{{% /notes %}}
+
+To view your access tokens, or create a new one, view the <a href="https://app.pulumi.com/account/tokens" target="_blank">Access Tokens</a> page. You will see a list of past tokens, when they were last used, and have the ability to revoke them.
+
+## Required Request Headers
+
+TODO: Add more info.
```

Comment:
FYI, the MDN documentation is great source for info on this.

For context, the `Content-Type` is kinda assumed, but good to mention. The `Accept` header is used to mark the version of the Pulumi API you are using. e.g. whenever we make a breaking change to a CLI endpoint we bump the `Accept` header newer CLIs send. (But will maintain backwards compatibility if we see an older `vhd.pulumi+{version}` number.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Perhaps not something you should mention, but technically you can omit the `Accept` header, and we will default to a ~recent version number. But it isn't recommended, as omitting the `Accept` header can lead to breaking changes (if we change the "default" version number used).